packages/edit-site/src/components/layout/index.js

```diff
@@ -294,12 +294,7 @@ export default function Layout() {
 					<SavePanel />
 
 					{ isMobileViewport && areas.mobile && (
-						<div
-							className="edit-site-layout__mobile"
-							style={ {
-								maxWidth: widths?.content,
```

Comment:
If widths become necessary for mobile, I think the router should declare them for this specific "area" the same it does for "content" — e.g.: `maxWidth: widths?.mobile`.

Reply:
So this change has no impact?

Reply:
Nope. This only affected to pages that whose mobile had content + had a width set (the existing logic only accounted for the list layout):

https://github.com/WordPress/gutenberg/assets/583546/2ea2ada3-b043-4ac7-86ff-653af63de0e1

Reply:
Was testing a different thing in `trunk` and this is what happened with this rule in place:


https://github.com/WordPress/gutenberg/assets/583546/c4fdc427-b796-4821-8777-cb2822d77305